app/controllers/recipes_controller.rb

```diff
@@ -1,28 +1,25 @@
 class RecipesController < ApplicationController
-
-
-before_filter :set_default_response_format
+  before_action :authenticate
 
   # GET /recipes
   # GET /recipes.json
   def index
-    @recipes = Recipe.all
-
-    render
+    @recipes = @current_user.recipes.load
+    render json: @recipes
   end
 
   # GET /recipes/1
   # GET /recipes/1.json
   def show
-    @recipe = Recipe.find(params[:id])
+    @recipe = @current_user.recipes.find(params[:id])
```

Comment:
This is a silly thing, but I'd prefer a reader method `current_user` rather than accessing an instance variable set in a superclass directly. I think it'd be easier to track down that way.